### ISSUE TITLE
copy symlinks created blobs fails when copying to another container

### ISSUE THREAD

**Issue by dwr-psandhu** [microsoft/AzureStorageExplorer#8127]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.34.0

### Regression From

N/A

### Architecture

x64

### Platform

Windows

### Bug Description

I used the --preserve-symlinks to copy from linux keeping the symbolic links as blobs. However these special blobs are not respected by storageexplorer when copying from one container to another. Downloads also simply fail to materialize a file on Windows. 

### Steps to Reproduce

1. Copy symbolic links from linux to a blob
2. Use storage explorer to copy these blobs to another container. This fails
3. Try to download it . It fails. 

### Actual Experience

Symbolic links disappear or are not copied over.

**Comment by craxal** [comment]
@dwr-psandhu
* How are you doing step 1? Are you able to successfully upload the symlinks using Storage Explorer?
* Are you uploading to a standard blob container or an ADLS Gen2 container?
* Are you trying to copy to a standard blob container or an ADLS Gen 2 container?
* Can you provide the exact error messages you are seeing?

**Comment by dwr-psandhu** [comment]
Step 1 is by using azcopy --preserve-symlinks
Both source and destination are Storage v2 general purpose
No error messages. Just the blob that represent symlinks are not copied over to destination. They are missing.

**Comment by craxal** [comment]
> Both source and destination are Storage v2 general purpose.

So hierarchical namespaces is _not_ enabled, correct?

> No error messages. Just the blob that represent symlinks are not copied over to destination. They are missing.

Interesting. Can you share both your app logs and your AzCopy logs? Can you copy the symlinks using AzCopy directly, and does it work as expected?

**Comment by dwr-psandhu** [comment]
Yes it says on the portal that "Hierarchical namespace  Disabled"
I am using azcopy command line from linux 10.24+ to copy symlinks. I can also use azcopy with --preserve-symlinks to copy between blob containers and that works fine. 
It is really storage explorer that doesn't have that option turned on, e.g. when i ask to see what azcopy command storage explorer uses it doesn't have "--preseve-symlinks" in it stated options.

**Comment by craxal** [comment]
@dwr-psandhu Correct, Storage Explorer is not using the `--preserve-symlinks` to copy. We will need to discuss the full implications of including this. We may just add an app setting to enable it.

**Comment by dwr-psandhu** [comment]
Ok, I believe it is a harmless option or perhaps there is another option that keeps metadata on blobs intact ?

**Comment by dwr-psandhu** [comment]
It is my understanding that when this feature was implemented it was as a blob with contents as name of the file being linked to and metadata property indicating it was a symlink
https://github.com/Azure/azure-storage-azcopy/pull/1829/commits/873c86ab7001210fea258ce119d8f85c3970eb76
